packages/mock-app/src/pages/Verify.tsx

```diff
@@ -95,17 +95,9 @@ const Verify = () => {
 
   const verifyCredential = async (verifiableCredential: VerifiableCredential) => {
     try {
-      const verifyCredentialParams = {
-        credential: verifiableCredential,
-        fetchRemoteContexts: true,
-        policies: {
-          credentialStatus: true,
-        },
-      };
-
       const verifyServiceUrl = appConfig.defaultVerificationServiceLink.href;
-      privateAPI.setBearerTokenAuthorizationHeaders(appConfig.defaultVerificationServiceLink.apiKey ?? '');
-      const verifiedCredentialResult = await privateAPI.post<IVerifyResult>(verifyServiceUrl, verifyCredentialParams);
+      const verifiedCredentialResult = await verifyVC(verifiableCredential, verifyServiceUrl);
+
```

Comment:
By default, the `verifyVC` function already retrieves the `appConfig.defaultVerificationServiceLink.href` from the `app-config.json`. So I don’t think we need to get the same `appConfig.defaultVerificationServiceLink.href` here.

Reply:
Hi @huynguyen-hl, this is an issue I have addressed in the latest commit to this branch, and this [PR](https://github.com/uncefact/tests-untp/pull/131) also sought to address.